azurerm/internal/services/servicebus/validate/sql_filter_test.go

```diff
@@ -0,0 +1,39 @@
+package validate
+
+import (
+	"strings"
+	"testing"
+)
+
+func TestSqlFilterValidateFunc(t *testing.T) {
+	cases := []struct {
+		Value       string
+		ShouldError bool
+	}{
+		{
+			Value:       "to='terraform'",
+			ShouldError: false,
+		},
+		{
+			Value:       "",
+			ShouldError: true,
+		},
+		{
+			Value:       strings.Repeat("user.foo='bar' AND ", 55)[:1040],
```

Comment:
I'm not sure of a better way to generate this string as we'll eventually want a grammar compliant test value, not just a long string